README.md

```diff
@@ -333,6 +333,42 @@ Assert that the mixin makes a declaration of the given rule-property pair.
 sassaby.includedMixin('appearance').calledWithArgs('button').declares('-webkit-appearance', 'button');
 ```
 
+#### declaresProperties
+Assert that the mixin makes a declaration of the given property with fallback values.
+E.g. for the mixin: 
+```
+@mixin font-size($font-size: 1.6) {
+    font-size: $font-size * 16 + px;
+    font-size: $font-size + rem;
+}
+```
+```js
+sassaby.includedMixin('font-size').calledWithArgs('2').declaresProperties('max-height', ['32px', '2rem']);
```

Comment:
I think I'd rather have this functionality in the current API. I could see `declares` taking either a string or an array. If it is an array with one item it will echo current behavior and if its an array with multiple items it will make the fallback assertion defined here.